rbac/management/role/view.py

```diff
@@ -36,6 +36,10 @@
 
 TESTING_APP = os.getenv('TESTING_APPLICATION')
 APP_WHITELIST = ['cost-management', 'catalog', 'approval']
+ADDITIONAL_FIELD_KEY = 'add_fields'
```

Comment:
If this can only be one field, maybe we should change the value to `add_field`. If we're able to support multiple fields, maybe changing the constant name to `ADDITIONAL_FIELDS_KEY` makes more sense.

Reply:
Changed